server/compose/types/record_detect_duplicates_test.go

```diff
@@ -355,11 +358,12 @@ func TestDedupRule_checkMultiValueEqualDuplication(t *testing.T) {
 					Set: []RecordValueError{
 						{
 							Kind:    deDupError.String(),
-							Message: locationRule.IssueMultivalueMessage([]string{"{\"coordinates\":[-6.7833479,20.3768206]}", "{\"coordinates\":[0.7833479,10.3768206]}"}),
+                            Message: locationRule.IssueMultivalueMessage(),
 							Meta: map[string]interface{}{
 								"field":         "location",
 								"dupValueField": "location",
 								"rule":          locationRule.String(),
+                                "value": "{\"coordinates\":[-6.7833479,20.3768206]}, {\"coordinates\":[0.7833479,10.3768206]}",
```

Comment:
what are these magic numbers?

Reply:
They are just test data co-ordinates

Reply:
sorry, mistakenly did not see it was a test file